src/Core_Language_Command.php

```diff
@@ -168,26 +168,26 @@ public function is_installed( $args, $assoc_args = array() ) {
 	/**
 	 * Installs a given language.
 	 *
-	 * Downloads the language pack from WordPress.org.
+	 * Downloads the language pack from WordPress.org. Find your language code at: https://translate.wordpress.org/
 	 *
 	 * ## OPTIONS
 	 *
 	 * <language>...
-	 * : Language code to install.
+	 * : Language code to install, it is based on the ISO-639 language code (e.g. pt for Portuguese) followed by the ISO-3166 country code (e.g. PT for Portugal or BR for Brazil). So, the Brazilian Portuguese file would be called pt_BR
```

Comment:
This explanation does accommodate the cases of same Language Code and Locale Code, `ja`, `ar`, etc.

Reply:
I think just linking to the website above is enough, no?

Reply:
I think it is enough, with the updated example and the url it should be fairly easy to identify a proper language code.

Reply:
Ok, then lets revert this explanation and add just link only in the description.

Reply:
Reverted it. Have kept the URL same, waiting for feedback.